Error when running linter locally

Try running `make lint` on your local system, and you'll get the following error:

```
level=warning msg="[runner] Can't run linter goanalysis_metalinter: bodyclose: failed prerequisites: [buildssa@github.com/hyperledger/aries-framework-go-ext/component/vdr/indy: analysis skipped: errors in package: [/opt/workspace/vdr.go:21:2: could not import github.com/hyperledger/indy-vdr/wrappers/golang/vdr (/go/pkg/mod/github.com/hyperledger/indy-vdr/wrappers/golang@v0.0.0-20201031155907-5f437d26ed71/vdr/cb.go:32:8: could not import C (cgo preprocessing failed))]]"
level=warning msg="[runner] Can't run linter unused: buildir: failed to load package vdr: could not load export data: no export data for \"github.com/hyperledger/indy-vdr/wrappers/golang/vdr\""
level=error msg="Running error: buildir: failed to load package vdr: could not load export data: no export data for \"github.com/hyperledger/indy-vdr/wrappers/golang/vdr\""
```

Note that this doesn't happen in the CI.

--- a/Makefile
+++ b/Makefile
@@ -17,4 +17,5 @@ license:
 
 .PHONY: lint
 lint:
-	@scripts/check_lint.sh
+	@EXCLUDE_LINT_PATH=./component/vdr/indy scripts/check_lint.sh
+	@LINT_PATH=./component/vdr/indy GOLANGCI_LINT_IMAGE="canislabs/golangci-lint:latest" scripts/check_lint.sh

